app.py

```diff
@@ -46,14 +48,22 @@ def right():
 @app.route("/automate_vehicle")
 def automate_vehicle():
     # TODO:: Instantiate the class and call required functions
-    initPoint = Coordinates('basic_science',(12.96626,77.71211))
-    finalPoint = Coordinates('ganesha_statue',(12.96598,77.71148))
-    automate_drive = Automation(initPoint,finalPoint)
+    initPoint = Coordinates('basic_science', (12.96626, 77.71211))
+    finalPoint = Coordinates('ganesha_statue', (12.96598, 77.71148))
+    automate_drive = Automation(initPoint, finalPoint)
     automate_drive.automate_linear()
     return "a"
 
 
+@app.route("/tree")
+def tree():
+    gt = GeocodeTree()
+    gt.add_tree([77.71111, 12.96026])
```

Comment:
Here we would have to call the get_gps_coordinate() instead of giving the point directly... because we just are pressing the button right